apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet/Trace/SimpleManagedTracer.cs

```diff
@@ -19,7 +19,9 @@
 using System.Collections.Generic;
 using System.Diagnostics;
 
-namespace Google.Devtools.AspNet
+using CloudTrace = Google.Cloud.Trace.V1.Trace;
```

Comment:
Right, that's unfortunate but hard to avoid. I wonder whether it would be worth using `TraceProto` as the name, to make it clear what you're referring to? This isn't public though, so we can do that later on.